website/frontend/src/common/components/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { TextareaHTMLAttributes } from 'react';
+import clsx from 'clsx';
+import styles from './Textarea.module.scss';
+
+interface Props extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  label?: string;
+}
+
+const Textarea = ({ label, className, rows = 5, ...attrs }: Props) => {
```

Comment:
could we think combine input and text area to one component [like here](https://github.com/gear-tech/gear-js/pull/499)?

Reply:
Something to think off